autopush/web/message.py

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
part of me prefers that, if nothing else than to show that there are no system includes in this file (and it even passes flake8 ;) ) Killing it anyways.